bangazon-clumsy-penguins/BangazonAPI
# Pp employee controller

# Pp employee controller

## Description

Controller to interact with Employee Entity. Get Functions will return an Employee Object with Embedded department and Computer Objects if Applicable for that Employee. 

New Employees can be added and existing Employees can be modified.

## Steps to Test

Start API Server

Run Get requests for both an entire list and with an Id specified in route.
Request a non existent Id to test exception handling.
Add a new Employee with POST method.
Update an Existing Employee with PUT.
Attempted to update non existent employee to ensure exception handling.

##### System configuration.

Windows 10

##### 3rd party libraries that need to be installed.
none
##### Command line utilities to run.
none
##### Link to Feature Ticket
#6
